p2panda-rs/src/schema/schema_id.rs

```diff
@@ -30,13 +31,20 @@ impl SchemaId {
             "schema_field_v1" => Ok(SchemaId::SchemaField),
             string => {
                 // We only use document_id in a relation at the moment.
-                let hash = Hash::new(string)?;
-                Ok(SchemaId::Application(Relation::new(hash, vec![])))
+                Ok(SchemaId::Application(Relation::new(DocumentId::new(
```

Comment:
Now that we have pinned relations `SchemaId::Application` should contain a `PinnedRelation`. As long as we don't have migrations we always want to refer to a specific version of a schema.

Reply:
Yes! I've added it in the PR description under "Next Steps", will make an issue as soon as all is good here :+1: